docs/registry.md

```diff
@@ -88,7 +88,7 @@ Below are some slightly redacted examples from the services that don't support t
 **Docker Hub**
 
 ```
-$ sudo ./out/soci push --user $USERNAME:$PASSWORD docker.io/####/busybox:latest
+$ sudo nerdctl push docker.io/####/busybox:latest
```

Comment:
We might want to leave this file alone because the output is the actual shell output I got when I tested these registries.

Reply:
My only concern is that leaving it in would potentially suggest to users that this is the preferred way of using the CLI when our preferred way is to not use it at all.

I can remove it, but I would definitely like to modify this doc to use nerdctl and show the proper expected output.

Reply:
If we're going to change the command, I'd like to actually run them and use real output.

Reply:
Makes sense. I can try and set up these registries and get the proper output, if that doesn't work out I'll revert this change.

Reply:
Will be reverting the changes. Would it perhaps be worth mentioning in the registry file that we generally suggest using `nerdctl push` instead of `soci push` as a disclaimer? Or is it niche enough that we don't need to bother?

Reply:
No, I don't think it's necessary. We can update these next time we test these registries.